components/function-controller/internal/webhook/conversion_webhook.go

```diff
@@ -216,10 +218,14 @@ func convertResourcesV1Alpha1ToV1Alpha2(in *serverlessv1alpha1.Function, out *se
 	}
 
 	if buildResourcesExists {
+		//TODO: remove ResourceConfiguration in next step
 		out.Spec.ResourceConfiguration.Build.Resources = &in.Spec.BuildResources
+		setV1Alpha2BuildJobResources(in.Spec.BuildResources, out)
 	}
 	if functionResourcesExists {
+		//TODO: remove ResourceConfiguration in next step
```

Comment:
Is this PR ready to review? 🤔

Reply:
Yes, this part will be removed after changes in controller which are part of the same issue.